docs/operate/control/headless-app.md

```diff
@@ -393,22 +393,9 @@ Select from the **Levels** dropdown menu to filter the logs by severity level:
 {{% /tab %}}
 {{% tab name="Command line" %}}
 
-{{< tabs >}}
```

Comment:
Sorry @Otterverse I noticed we have https://docs.viam.com/operate/control/headless-app/#debug - it looks like journalctl and viam.log doesn't contain much on my machines. Is there a use case here or should I just point people at the CLI?

Reply:
I'm not sure I get the context here. The "viam-server" systemd unit only exits for the obsolete AppImage style installs, and logs would be under the "viam-agent" unit/service name on machines installed with the recommend (agent) method.

Also, viam.log would likely only show up if explicitly running it as a homebrew service on a Mac.

I don't think this information is critical here (so fine to remove), as we have it in the page on/for agent already.

Reply:
I can't follow a link to "localhost" (that's your personal laptop/desktop) but looking at the commits since my last review, I think those look fine.

Reply:
Sorry - I did mean to send https://deploy-preview-4457--viam-docs.netlify.app/manage/troubleshoot/troubleshoot/#machine-shows-as-offline but merging now - thanks!